src/Tizen.NUI/src/public/Images/EncodedImageBuffer.cs

```diff
@@ -0,0 +1,151 @@
+        }
+
+        internal EncodedImageBuffer(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemoryOwn)
+        {
+        }
+
+        /// <summary>
+        /// Generate URI from current buffer.
+        /// We can now use this url for ImageView.ResourceUrl
+        /// Note : the url lifecycle is same as ImageUrl and it's internal usage.
+        /// Store only ImageUrl.ToString() result and re-use that url is Undefined Behavior.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public ImageUrl GenerateUrl()
+        {
+            mCachedImageUrl ??= new ImageUrl(Interop.EncodedImageBuffer.GenerateUrl(this.SwigCPtr.Handle), true);
+            return mCachedImageUrl;
+        }
+
+        /// <summary>
+        /// Get current raw buffer. (read-only)
+        /// Note : the raw buffer doesn't have memory ownership.
+        /// Access to write something to raw buffer is Undefined Behavior.
+        /// </summary>
+        /// This will not be public opened.
+        internal VectorUnsignedChar GetRawBuffer()
+        {
+            VectorUnsignedChar ret = new VectorUnsignedChar(Interop.EncodedImageBuffer.GetRawBuffer(this.SwigCPtr.Handle), false);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            return ret;
+        }
+
+        /// <summary>
+        /// Dispose
+        /// </summary>
+        /// <param name="type"></param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected override void Dispose(DisposeTypes type)
+        {
+            if (disposed)
+            {
+                return;
+            }
+
+            if (type == DisposeTypes.Explicit)
+            {
+                //Called by User
+                //Release your own managed resources here.
+                //You should release all of your own disposable objects here.
+                mCachedImageUrl?.Dispose();
+            }
+
+            base.Dispose(type);
+        }
+
+        /// This will not be public opened.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected override void ReleaseSwigCPtr(HandleRef swigCPtr)
+        {
+            Interop.EncodedImageBuffer.DeleteEncodedImageBuffer(swigCPtr);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        /// <summary>
+        /// Get VectorUnsignedChar from System.IO.Stream.
+        /// This funcion is private (exist only for Constructor.)
+        /// </summary>
+        private static VectorUnsignedChar GetRawBuffrFromStreamHelper(System.IO.Stream stream)
+        {
+            if(stream == null)
+            {
+                throw new ArgumentNullException(nameof(stream));
+            }
+            long streamLength = stream.Length - stream.Position;
```

Comment:
Do not use `Position` property, Only specific streams support it
https://docs.microsoft.com/en-us/dotnet/api/system.io.stream.position?view=net-5.0#remarks

Reply:
Fix done. Thank you for catch my mistake.
I make same mistake at `ImageLoading.cs`. I'll fix it another PR